poc_mobile_verifier/src/heartbeats.rs

```diff
@@ -193,23 +188,30 @@ impl Heartbeat {
 
         Ok(sqlx::query_as::<_, HeartbeatSaveResult>(
             r#"
-            insert into heartbeats (hotspot_key, cbsd_id, reward_weight, timestamp)
-            values ($1, $2, $3, $4)
-            on conflict (cbsd_id) do update set
-            hotspot_key = EXCLUDED.hotspot_key, reward_weight = EXCLUDED.reward_weight, timestamp = EXCLUDED.timestamp
```

Comment:
I believe this will leave the hotspot under the old hotspot_key instead of moving it to the new hotspot_key in the case of a moved radio.  Also in that scenario, should we blank out all the timestamps?

Also, shouldn't we update the reward_weight still?